message_converter/src/converter_node.cpp

```diff
@@ -193,7 +136,48 @@ void SwiftNavRover::publishBaselineVelocity(ros::Time t, int tow,double n,double
     msg.covariance = covariance;
     msg.vel_mode = vel_mode;
     msg.ins_mode = ins_mode;
-    ned_vel_cov_fix.publish(msg);
+    ned_vel_cov_fix_.publish(msg);
+}
+
+void SwiftNavRover::publishRTKAvailable()
+{
+    std_msgs::Bool msg;
```

Comment:
@ssonawane-humatics the RTK availability message type should also have a time stamp associated with it. Will have to create a custom message for this.

FYI @pvazteixeira-humatics

Reply:
New message type added in https://github.com/Humatics/ethz_piksi_ros/pull/1/commits/398f00acf0b225459f23393d5c03329c25f94e21

This message also needs to be updated in ```le_msgs``` repo, will open a pull request in that repository shortly